### PR TITLE
refactor: return DataFrame type from read functions and None from writing ones

### PR DESCRIPTION
**Pull request by chamini2**

### CODE REVIEW

**Comment by chamini2** [comment]
https://github.com/fal-ai/fal/runs/7114868311?check_suite_focus=true

**Comment by chamini2** [comment]
Ignore failing test, I will merge the fix from #425

**Review by mederka — APPROVED** [review]

**Comment by chamini2** [comment]
https://github.com/fal-ai/fal/actions/runs/2591039422

File: src/faldbt/project.py [hunk 1]
```diff
@@ -755,32 +751,26 @@ def execute_sql(self, sql: str) -> pd.DataFrame:
...
         # this workaround.
-        query_result = lib.execute_sql(
-            project_dir=self.project_dir,
-            profiles_dir=self.profiles_dir,
-            sql=compiled.compiled_sql,
-            profile_target=self._profile_target,
+        return lib.execute_sql(
+            self.project_dir,
+            self.profiles_dir,
+            self._profile_target,
```
[the remark this answers is not in the document]
Comment (chamini2) [reply src/faldbt/project.py:769]: Consistency, mostly

**Pull request merged**